packages/react-menu/src/components/MenuPopover/useMenuPopover.ts

```diff
@@ -90,7 +90,10 @@ export const useMenuPopover = (props: MenuPopoverProps, ref: React.Ref<HTMLEleme
 
   return {
     ...state,
```

Comment:
Please check this code few line above, something going wrong there:

![image](https://user-images.githubusercontent.com/14183168/130789752-32d66c01-1b5d-4892-b25b-2d142d964e87.png)

- `props` are spreaded to `state`
- handlers are used from `state`